ui/src/gear/hood/types.ts

```diff
@@ -201,6 +203,18 @@ export interface Pike {
    * dead - app is not running
    */
   zest: "live" | "dead" | "held";
+  /**
+   *  {@link Weft}s associated with this Pike
+   */
+  wefts: Weft[];
+  /**
+   *  {@link Perm}s granted to this Pike
```

Comment:
Are these accurate descriptions for `perms.pike` and `lacks.pike`?

Reply:
Yes.